lib/sidekiq_alive/worker.rb

```diff
@@ -1,18 +1,30 @@
+require "sidekiq/api"
 module SidekiqAlive
   class Worker
     include Sidekiq::Worker
-    sidekiq_options retry: false
+    sidekiq_options retry: false, queue: SidekiqAlive.config.queue_with_variant
 
     def perform
       write_living_probe
-      self.class.perform_in(SidekiqAlive.time_to_live / 2)
+      clean_old_queues
+      self.class.set(queue: config.queue_with_variant).perform_in(config.time_to_live / 2)
+    end
+
+    def clean_old_queues
```

Comment:
this should only be executed if is multi instance. We do not want to remove `default` queue
Is it necessary to remove the queue?
the most probable is that the job is in `scheduled` instead of the queue itself